### PR TITLE
Create a setup compatible with viewdocs.io for docs

### PR DESCRIPTION
**Pull request by erran-r7**
You can preview the docs on [viewdocs.io](http://rapid7.viewdocs.io/convection~viewdocs/getting-started/).

* Update link to getting started guide in README.md
* Copy README.md to docs/index.md for now
  We can expand on this later.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- To print out a list of available cli options with their descriptions run `convection help`.
+
+###### Print
+- To print out the cloud formation template for a specific stack run `convection print my-stack-name`.
+
+###### Validate
+- To validate your stack is not missing a required resource run `convection validate my-stack-name`.
+
+## Documentation
+We highly recommend consulting the [getting started guide](./docs/getting-started.md) for a in depth walk through on how to to set up your project and create and deploy a stack. Example stacks and resources are available in the [convection/example](https://github.com/rapid7/convection/tree/master/example) folder
```
Comment (erran-r7) [inline docs/index.md]: Pretty sure this wouldn't resolve quite right. 😓

Comment (erran-r7) [inline docs/index.md]: Pretty sure this wouldn't resolve quite right. 😓

Comment (erran-r7) [inline docs/index.md]: I stand corrected, this resolves completely okay. 👍 Go viewdocs.

**Comment by bcall-r7** [comment]
Ship it

**Pull request merged**